schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "name": "hgvs",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_aggregated_interpretation_set",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "interpretation",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "type_aggregated_interpretation",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "type_single_interpretation",
+            "datatype": "string"
+          },
+          {
+            "name": "description_history",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "condition_list",
+            "datatype": "string"
+          },
+          {
+            "name": "number_of_submitters",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_of_submissions",
+            "datatype": "integer"
+          }
+        ]
+      },
+      {
+        "name": "type_single_interpretation",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "description",
+            "datatype": "string"
+          },
+          {
+            "name": "explanation",
+            "datatype": "string"
+          },
+          {
+            "name": "explanation_of_interpretation",
+            "datatype": "string"
+          },
+          {
+            "name": "xref",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "citation",
+            "datatype": "string"
+          },
+          {
+            "name": "comment",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "date_last_evaluated",
+            "datatype": "date"
+          },
+          {
+            "name": "allele_origin",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_citation",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "id",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "url",
+            "datatype": "string"
+          },
+          {
+            "name": "citation_text",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          },
+          {
+            "name": "abbrev",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "id",
```

Comment:
This feels confusing enough to justify a special-case...

Talked offline, and this particular use of `ID` occurs under the `citation` table. We could move `source` into `citation` as a column, and eliminate this table.